templates/home/index.html

```diff
@@ -342,4 +336,6 @@ <h1 data-aos="fade-up">Contact Us</h1>
                      
                     
                   
-                
\ No newline at end of file
+                            
+                
+    
```

Comment:
So, I asked you to remove the lines at the end of file but instead you added two more lines. Why so?